README.md

```diff
@@ -7,24 +7,26 @@ Terraform-Qingcloud-Plugin [![Build Status](https://travis-ci.org/yunify/terrafo
 
 ## Usage
 
-### Install qingcloud-provider
-
-#### On Linux
-``` bash
-go get github.com/yunify/terraform-provider-qingcloud
-glide up
-make build
-make test
-cp ./terraform-provider-qingcloud $(dirname `which terraform`)/terraform-provider-qingcloud
-```
+### Install terraform-provider-qingcloud
+
```

Comment:
the official documents indicates that the plugins should be placed in third party plugin directory. https://www.terraform.io/docs/configuration/providers.html#third-party-plugins